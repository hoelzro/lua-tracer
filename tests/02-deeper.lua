local function foo(a)
  local b = a -- trace: a
end

local function bar(a)
  foo(a)
end

local function baz(a)
  bar(a)
end

baz(2)
bar(4)
