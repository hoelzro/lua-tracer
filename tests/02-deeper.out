a	2
a	4
